### ISSUE TITLE
Inception Model Error

### ISSUE THREAD

**Issue by Davidnet** [NVIDIA-Jetson/tf_to_trt_image_classification#1]
Trying the readme, I encounter:

```
UFFParser: Validator error: InceptionV1/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
```

**Comment by Davidnet** [comment]
In fact:

```
UFFParser: Validator error: MobilenetV1/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: resnet_v1_50/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: MobilenetV1/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: InceptionV2/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: InceptionV3/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: resnet_v2_152/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: InceptionV1/Logits/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: resnet_v1_152/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: InceptionV4/Logits/PreLogitsFlatten/flatten/Reshape: Unsupported operation Flatten
Failed to parse UFF
UFFParser: Validator error: resnet_v1_101/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
UFFParser: Validator error: InceptionResnetV2/Logits/Flatten/flatten/Reshape: Unsupported operation Flatten
Failed to parse UFF
UFFParser: Validator error: resnet_v2_50/SpatialSqueeze: Unsupported operation Squeeze
Failed to parse UFF
```

**Comment by jaybdub-nv** [comment]
Hi David,

The squeeze operation is supported in the version of TensorRT shipped with the upcoming JetPack 3.2 release (the JetPack 3.2 _developer preview_ does not contain this fix). 

John
